crds/embedded/cdi.yaml

```diff
@@ -1337,4 +1337,6 @@ spec:
           type: object
       served: true
       storage: true
-      subresources: {}
```

Comment:
Is it normal if we delete this fragment?

Reply:
Fixed